CaseDocs/atm_in_0001

```diff
@@ -15,7 +15,6 @@
 &cam_history_nl
  empty_htapes		= .true.
  fincl1		= 'PHIS:I'
- fincl2		= 'SST:I'
```

Comment:
Why was SST:I deleted here?

Reply:
Tim requested the SST:I for a previous assimilation
so that we could see what SST CAM was actually using.
Those issues have been resolved, so I'm simplifying
the run and archiving by no longer writing out the 
CAM 'h1' history file with SST in it.